snippets/RGBToHSV.md

```diff
@@ -0,0 +1,44 @@
+---
+title: RGBToHSV
+tags: math, intermediate
+---
+
+Converts the values of RGB color components to HSV components.
+
+Convert given RGB parameters into HSV ones and returns it in array format. 
```

Comment:
The new format suggests to use bullet points. I think you should:

- Explain the ranges of the values in one bullet point.
- Link to a wikipedia page of the calculation and/or format as reference.